### PR TITLE
Allow missing weights in row construction.

### PR DESCRIPTION
**Pull request by femtomc**

### CODE REVIEW

**Comment by hannahilea** [comment]
Test needed!

**Comment by femtomc** [comment]
@hannahilea / @ericphanson what sort of test would work here? Would construction with `missing` weights be appropriate?

**Comment by hannahilea** [comment]
yep!

**Comment by ericphanson** [comment]
I think we also don't want to check in `test/my_model.model.arrow`

**Comment by femtomc** [comment]
oops, can I add that to `.gitignore` (?)

**Comment by ericphanson** [comment]
sure!

File: test/runtests.jl [hunk 1]
```diff
@@ -21,6 +21,10 @@ end
     my_model = make_my_model()
     load_weights(my_model, test_weights())
 
+    # quick test with `missing` weights.
+    model_row = ModelRow(; weights=missing)
+    write_model_row("my_model.model.arrow", model_row)
```
Comment (ericphanson) [inline test/runtests.jl:26]:
```suggestion
    model_row = ModelRow(; weights=missing)
    write_model_row("my_model.model.arrow", model_row)
    rt = read_model_row("my_model.model.arrow")
    @test isequal(model_row, rt)
```

**Review by ericphanson — COMMENTED** [review]
Hm, I added reading it back, and the test failed 🤔. I guess we are assuming it is non-empty somewhere we shouldn't be- this might need a few more changes actually

**Comment by femtomc** [comment]
Interesting -- I think it depends on what we want semantically. It seems like adding `missing` as an element fixes the issue. I'm not exactly sure if that's good or bad.

**Comment by ericphanson** [comment]
I think #15 might be a simpler approach

**Comment by femtomc** [comment]
Closing in lieu of #15.

**Pull request closed without merging**